opam

```diff
@@ -16,37 +16,18 @@ maintainer: "Daniel Bünzli <daniel.buenzl i@erratique.ch>"
 authors: "The uucp programmers"
 license: "ISC"
 tags: ["unicode" "text" "character" "org:erratique"]
-homepage: "https://erratique.ch/software/uucp"
-doc: "https://erratique.ch/software/uucp/doc/"
+homepage: "git://github.com/dune-universe/uucp"
 bug-reports: "https://github.com/dbuenzli/uucp/issues"
 depends: [
   "ocaml" {>= "4.03.0"}
-  "ocamlfind" {build}
-  "ocamlbuild" {build}
-  "topkg" {build & >= "1.0.3"}
-  "uucd" {with-test}
-  "uunf" {with-test}
-  "uutf" {with-test}
+  "dune" {>= "1.1"}
```

Comment:
Is this not missing the dependencies on `uutf`, `uunf` and `cmdliner`?

Reply:
I think those packages are just used for `uucp`'s tests. I've tried building this port on its own and as a dependency in a monorepo and it builds fine without them. Should we keep them anyway so the tests can be run?

Reply:
Nah, that's fine. No need to run the tests in a monorepo context.